packages/email-field/test/email-field.test.js

```diff
@@ -79,4 +77,28 @@ describe('email-field', () => {
       expect(emailField.pattern).to.equal('.+@example.com');
     });
   });
+
+  describe('invalid', () => {
```

Comment:
Did you miss testing these cases previously? Or they came up after the changes in this PR?

Reply:
I missed adding these tests as they were present in other test suite (check out the corresponding test suite for vaadin-text-field package).